sympy/solvers/tests/test_ode.py

```diff
@@ -2573,7 +2576,7 @@ def test_nth_order_linear_euler_eq_nonhomogeneous_variation_of_parameters():
 
 def test_issue_5095():
     f = Function('f')
-    raises(ValueError, lambda: dsolve(f(x).diff(x)**2, f(x), 'separable'))
+    #raises(ValueError, lambda: dsolve(f(x).diff(x)**2, f(x), 'separable'))
```

Comment:
Working, I just forgot to remove #

Reply:
Are you going to remove the `#`?

Reply:
Does this test pass?

Reply:
No, now it can classify by `classify_ode`.